### PR TITLE
Use gunicorn as the application server and nginx as the web server

### LINKED ISSUE #15: Add gunicorn and nginx to docker-compose services for deployment
The idea is to start using gunicorn and nginx for deployment from the initial stages instead of relying on django development server

### PR DESCRIPTION
**Pull request by GoelJatin**
#15

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -16,11 +14,20 @@ services:
     env_file: .env.development
     volumes:
       - ./src:/app
-    ports:
```
Comment (inovizz) [inline docker-compose.yml:19]: We don't need to export port now?

**Comment by inovizz** [comment]
Looks good to me. 👍🏻

File: docker-compose.yml (hunk 1 above)
Comment (GoelJatin) [reply docker-compose.yml:19]: The port is exposed from the `Dockerfile.django`, we don't need to add the port forwarding from the physical machine to the container as the django app will only be reachable via the nginx server
I've changed the nginx port to run on port 80 so after building the services they can directly go to http://localhost to access the app

**Comment by GoelJatin** [comment]
merging to master

**Pull request merged**